app/src/main/java/org/simple/clinic/login/pin/LoginPinScreen.kt

```diff
@@ -80,6 +77,15 @@ class LoginPinScreen(context: Context, attrs: AttributeSet) : RelativeLayout(con
       RxView.clicks(backButton)
           .map { PinBackClicked() }
 
+  private fun otpReceived() =
+      Single
+          .fromCallable {
+            val key = screenRouter.key<LoginPinScreenKey>(this)
+            key?.otp ?: ""
+          }
+          .map { LoginPinOtpReceived(it) }
+          .toObservable()
+
```

Comment:
I'm not going to ask you to change this because this is another throwaway piece of code, but this could have been reduced to 2 lines :p 

```
val key = screenRouter.key<LoginPinScreenKey>(this)
return Observable.just(LoginPinOtpReceived(key!!.otp))
```